mlrun/api/schemas/frontend_spec.py

```diff
@@ -33,3 +44,4 @@ class FrontendSpec(pydantic.BaseModel):
     auto_mount_type: typing.Optional[str]
     auto_mount_params: typing.Dict[str, str] = {}
     default_artifact_path: str
+    default_user_pod_resources: typing.Optional[Resources]
```

Comment:
If the struct is well defined I don't see a reason not to include it
```suggestion
    default_function_pod_resources: Resources = Resources()
```
Do the same for `Resources` and for `Resource` give a default value of `None`